crates/ruma-html/src/html.rs

```diff
@@ -9,37 +9,37 @@ use html5ever::{
 };
 use tracing::debug;
 
-/// An HTML fragment.
+/// A HTML document.
 ///
 /// To get the serialized HTML, use its `Display` implementation.
 #[derive(Debug)]
-pub struct Fragment {
+pub struct Html {
```

Comment:
Maybe `HtmlDocument` to be more explicit?

Reply:
The reason we were using `Fragment` is that it is not considered a whole document. A document is supposed to have a doctype, and a `<head>` and `<body>`. But I thought that `Fragment` doesn't mean anything for people in general, that's why I renamed it here.

If you don't like just `Html`, maybe `HtmlFragment` then?

Reply:
You changed the documentation to say document, that's why I thought this made sense ^^

I think `Html` is a fine name, but let's change the docs back then?